Mathlib/Algebra/Category/Grp/IsFinite.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.Algebra.Category.Grp.Abelian
+import Mathlib.Algebra.Category.Grp.EpiMono
+import Mathlib.Algebra.Homology.ShortComplex.Ab
+import Mathlib.CategoryTheory.Abelian.SerreClass.Basic
+import Mathlib.Data.Finite.Prod
+
+/-!
+# The Serre class of finite abelian groups
+
+In this file, we define `isFinite : ObjectProperty Ab` and show
+that it is a Serre class.
+
+-/
+
+universe u
+
+open CategoryTheory Limits ZeroObject
+
+namespace Ab
```

Comment:
I think so far we have been putting results about the category of abelian groups into the namespace `AddCommGrp`.

Reply:
Thanks for the suggestions!